frontend/src/helpers/file/parse-file/parse-file.ts

```diff
@@ -0,0 +1,6 @@
+const parseFile = (payload: FileList | null): File => {
```

Comment:
don't forget to change the filename
```suggestion
const getFileFromFileList = (payload: FileList | null): File => {
```